src/rpc/common/impl/Processors.h

```diff
@@ -31,21 +31,61 @@ struct DefaultProcessor final
     operator()(HandlerType const& handler, boost::json::value const& value)
         const
     {
-        using boost::json::value_from;
-        using boost::json::value_to;
+        // can not do the check in one if statement
+        // first check the nested type exist
+        // then check has a process function called by this type
+        constexpr bool hasNonVoidInputField =
+            requires(const HandlerType& t) { typename HandlerType::Input; };
+        if constexpr (hasNonVoidInputField)
+        {
+            constexpr bool hasProcessWithInput =
```

Comment:
and here too.. seems like unnecessary code duplication.

Reply:
I can not put them in one if . seems it does not support short circuit